### DISCUSSION TITLE
Move mute info icons to user label

### DISCUSSION THREAD

**Discussion by Hk1020** [jamulussoftware/jamulus#1251] [Ideas]
I've just been experimenting with three local clients to understand what the mute info icons above the faders actually do and I found it quite confusing. The icons appear if for some reason the person of that particular fader can't hear you because he soloed or muted something. It is out of my control. Everything else, however, affects and shows only what **I** hear.

I would find it much more logical to move this info icon into the user label area. There I see info about the remote user, like their name. What I see above is stuff that I do.

What do you think? There is a related issue #811 which would be affected by such a change.

I'd also like to see my name somewhere in the GUI like in the title bar or as `Joe's personal mix at: _server_` so I know which fader is mine.

**Comment by pljones** [comment]
I agree that the current location is confusing.  The "speaker icon struck through" looks like I'm not including them in my mix.  An "ear icon struck through" next to the person's name would more clearly indicate that person isn't listening to me.

**Comment by gilgongo** [comment]
We're currently limited to using Unicode characters for the icons I think. But the part of this ticket that related to the icon itself might also be relevant to https://github.com/jamulussoftware/jamulus/issues/899

Mind you, icons on their own are always a bit iffy - if there was space to put "Cannot year you" in text somewhere that would be best, but there may not be in the current layout:

![image](https://user-images.githubusercontent.com/4561747/106524631-c1159f80-64da-11eb-8bd0-db975b48d1aa.png)

**Comment by pljones** [comment]
Accessibility should definitely be a consideration, too.  Hover text / alt-text on all icons is always beneficial.

I run using compact skin... so you could end up with "smaller than small" text, probably less than readable.

**Comment by gilgongo** [comment]
Yes, hence my doubts about the space. 

BTW I must admit I don't know much about screen readers and desktop apps. Do they parse alt text like they do with web browsers?

**Comment by pljones** [comment]
> BTW I must admit I don't know much about screen readers and desktop apps. Do they parse alt text like they do with web browsers?

AFAIK, yes - I don't know if you'd need a Qt-specific one or if Qt marks up from the accessibility text with some common standard.  I just generally consider it good practice.

I guess it could be tested with at least the Windows screen reader.  I don't know on MacOS or Linux.

**Comment by Hk1020** [comment]
My idea was to have something actually right in the user label itself, i.e., the white, red or green areas in your screenshot.

**Comment by gilgongo** [comment]
Yes, although there's not currently room for much at all in there (and text would be clearer). 

There are a few things that might be accumulating that point toward a bit of a re-arrangement of the UI, I think. This ticket being one of them.
